cerberus-api-tests/build.gradle

```diff
@@ -0,0 +1,90 @@
+import com.github.spotbugs.snom.SpotBugsReport
```

Comment:
In order for Dependabot to work files need to be named "build.gradle" this isn't a new file, just a rename of cerberus-api-tests.gradle.